ctru-rs/src/services/cfgu.rs

```diff
@@ -0,0 +1,212 @@
+//! Configuration service
+//!
+//! This module contains basic methods to retrieve and change configuration from the console.
+
+#[derive(Copy, Clone, Debug)]
+#[repr(u32)]
+pub enum Region {
+    Japan = ctru_sys::CFG_REGION_JPN,
+    USA = ctru_sys::CFG_REGION_USA,
+    Europe = ctru_sys::CFG_REGION_EUR,
+    Australia = ctru_sys::CFG_REGION_AUS,
+    China = ctru_sys::CFG_REGION_CHN,
+    Korea = ctru_sys::CFG_REGION_KOR,
+    Taiwan = ctru_sys::CFG_REGION_TWN,
+}
+
+#[derive(Copy, Clone, Debug)]
+#[repr(u32)]
+pub enum Language {
+    Japanese = ctru_sys::CFG_LANGUAGE_JP,
+    English = ctru_sys::CFG_LANGUAGE_EN,
+    French = ctru_sys::CFG_LANGUAGE_FR,
+    German = ctru_sys::CFG_LANGUAGE_DE,
+    Italian = ctru_sys::CFG_LANGUAGE_IT,
+    Spanish = ctru_sys::CFG_LANGUAGE_ES,
+    SimplifiedChinese = ctru_sys::CFG_LANGUAGE_ZH,
+    Korean = ctru_sys::CFG_LANGUAGE_KO,
+    Dutch = ctru_sys::CFG_LANGUAGE_NL,
+    Portuguese = ctru_sys::CFG_LANGUAGE_PT,
+    Russian = ctru_sys::CFG_LANGUAGE_RU,
+    TraditionalChinese = ctru_sys::CFG_LANGUAGE_TW,
+}
+
+#[derive(Copy, Clone, Debug)]
+#[repr(u32)]
+pub enum SystemModel {
+    Model3DS = ctru_sys::CFG_MODEL_3DS,
+    Model3DSXL = ctru_sys::CFG_MODEL_3DSXL,
+    ModelNew3DS = ctru_sys::CFG_MODEL_N3DS,
+    Model2DS = ctru_sys::CFG_MODEL_2DS,
+    ModelNew3DSXL = ctru_sys::CFG_MODEL_N3DSXL,
+    ModelNew2DSXL = ctru_sys::CFG_MODEL_N2DSXL,
+}
+
+/// Represents the configuration service. No actions can be performed
+/// until an instance of this struct is created.
+///
+/// The service exits when all instances of this struct go out of scope.
+pub struct Cfgu(());
+
+impl Cfgu {
+    /// Initializes the CFGU service.
+    ///
+    /// # Errors
+    ///
+    /// This function will return Err if there was an error initializing the
+    /// CFGU service.
+    ///
+    /// ctrulib services are reference counted, so this function may be called
+    /// as many times as desired and the service will not exit until all
+    /// instances of Fs drop out of scope.
```

Comment:
Replace "Fs" with "Cfgu"

Reply:
Also, if you are referencing a real item that's in scope, you can write ``[`Cfgu`]`` for generated docs to make a clickable link to the actual `Cfgu` struct, which isn't super relevant here since it's already under the impl, but it looks nice.

Reply:
> Replace "Fs" with "Cfgu"

Oops, i used the Fs documentation as a reference an looks like I copy-pasted that and forgot to change it :p